### PR TITLE
Update design of plan step from Sensei onboarding

### PR DESCRIPTION
**Pull request by renatho**
## Proposed Changes

* It updates the Sensei onboarding plan design, in order to match other flows.
* It doesn't change any functionality.

## Considered Alternatives

* I started the working trying to implement it tied to the Calypso code, similar to https://github.com/Automattic/wp-calypso/pull/75174 and https://github.com/Automattic/wp-calypso/pull/76358.
* After some work, I realized that we have some different integrations that seemed a little complex to port to the mentioned approach. So, after talking with Zen to collect previous experiences, I decided to keep the current approach: try to use the available flexible components from Calypso and tweak some styles to match the other flows.
* Maybe in the future we'll want to invest more time trying to use the mentioned approach with our custom integrations. So it would use the same components as the other flows.

## Testing Instructions

* Navigate to `/setup/sensei/senseiPlan`.
* Make sure it works properly.

I asked for design and texts review here: p1684871185282869-slack-C048ZJB2QNM

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

## Screenshots

Screenshot 2023-05-23 at 15 53 54

Screenshot 2023-05-23 at 15 54 03

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/styles.scss [hunk 1]
```diff
@@ -149,4 +177,13 @@
 			margin-bottom: 0;
 		}
 	}
+
+	.components-popover {
```
Comment (renatho) [inline client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/styles.scss:181]: For some reason, the popover was with the arrow white. Maybe some update broke that. These styles fix that.

**Comment by burtrw** [comment]
Can we change "Choose Monthly or Annually" to "There’s a plan for you."

This is what the other flows use :)

**Comment by renatho** [comment]
> Can we change "Choose Monthly or Annually" to "There’s a plan for you."
> 
> This is what the other flows use :)

Sure! Updated here: https://github.com/Automattic/wp-calypso/pull/77276/commits/16c9ceb08ff59eaf5495dba2bc658f1604a42146

**Review by aaronfc — APPROVED** [review]
Looks good and works well :)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8027788

Thank you @renatho for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8027788).

**Pull request merged**